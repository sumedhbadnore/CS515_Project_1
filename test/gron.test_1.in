{
  "person": {
    "name": "Alice",
    "age": 25,
    "address": {
      "city": "Paris",
      "country": "France"
    }
  }
}