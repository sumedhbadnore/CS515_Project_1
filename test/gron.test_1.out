json = {};
json.person = {};
json.person.name = "Alice";
json.person.age = 25;
json.person.address = {};
json.person.address.city = "Paris";    
json.person.address.country = "France";